games-roguelike/dwarf-fortress/dwarf-fortress-0.47.04.ebuild

```diff
@@ -31,8 +31,8 @@ RDEPEND="media-libs/glew:0
 DEPEND="${RDEPEND}
 	media-libs/libsndfile
 	media-libs/openal
-	sys-libs/ncurses-compat:5[unicode]
-	virtual/pkgconfig"
+	sys-libs/ncurses-compat:5[unicode]"
```

Comment:
I know you haven't changed this but `ncurses-compat` in `DEPEND` doesn't make any sense. This package does not install any headers. I can see that the sources are hardcoded to load `libncurses.so.5`. This is really bad if you're building with the headers from `libncurses.so.6`. I can see that it falls back to `libncurses.so` but that's no use if it finds `.so.5` first. The fact that it builds means it must be compatible with `.so.6` so I guess you should patch `renderer_curses.cpp` accordingly. This whole thing is really horrible though.

Reply:
Let's see what I can find. In the worst case it looks like this:
https://i.imgur.com/Kx4NvI4.gif